<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pButton2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Review</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>84</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="cBoxOne">
      <property name="text">
       <string>One</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="cBoxTwo">
      <property name="text">
       <string>Two</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="cBoxThree">
      <property name="text">
       <string>Three</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="cBoxFour">
      <property name="text">
       <string>Four</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="cBoxFive">
      <property name="text">
       <string>Five</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
